### DISCUSSION TITLE
Batch size limit?

### DISCUSSION THREAD

**Discussion by M-Walton** [adieyal/sd-dynamic-prompts#285] [Q&A]
It seems I can produce a single image just fine, but if I run a batch, a lot of the time I come back and all of the prompts are the raw input instead of processed prompts. I managed to run a batch count of 20 and it worked as expected, but I try again and it does not.

Usually I like to create a dynamic prompt with a lot of varied elements, and leave a batch of 100 running, and come back to see what surprises it has generated for me. I was doing this on older versions of the UI and library last week, but just installed the latest and I have hit this snag. Is there some setting I need to change to make this work? Am I doing something wrong?

using version 2.7.3 of the WebUI extension, and the underlying dynamicprompts library is version 0.7.1.

**Comment by adieyal** [comment]
Any errors in the console?
